main_test.go

```diff
@@ -1,11 +1,82 @@
-package main
+package main_test
 
 import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/matryer/is"
+	proj "github.com/sheldonhull/go-semantic-sentences"
+	zl "github.com/sheldonhull/go-semantic-sentences/internal/logger"
 )
 
-func countViolations(t *testing.T) {
+// Logger contains the package level logger provided from internal logger package that wraps up zerolog.
+// var Log *logger.Logger //nolint: gochecknoglobals.
+//nolint
+func TestMain(m *testing.M) {
+	c := zl.Config{
```

Comment:
I can get rid of this now since I added the `init` to ensure the logger is initialized, but set to disabled.